package.json

```diff
@@ -0,0 +1,46 @@
+{
+  "name": "type-head-input",
+  "version": "0.0.0",
+  "scripts": {
+    "ng": "ng",
+    "start": "ng serve",
+    "build": "ng build",
+    "build-lib": "ng build typeHeadInput",
+    "pack-lib": "cd dist/type-head-input && npm pack",
+    "watch": "ng build --watch --configuration development",
+    "test": "ng test"
+  },
+  "private": true,
+  "dependencies": {
+    "@angular/animations": "^15.1.4",
```

Comment:
@sachith94-lion can't we move to latest versions?

Reply:
@DevDHera Currently, I added what in bsb v2 project used is it ok to update into 17?